requestium/requestium.py

```diff
@@ -89,6 +90,18 @@ def _start_chrome_browser(self):
             else:
                 raise Exception('A list is needed to use \'arguments\' option. Found {}'.format(
                     type(self.webdriver_options['arguments'])))
+                
+        # adds in ability to set a new download directory
+        # https://stackoverflow.com/questions/35331854/downloading-a-file-at-a-specified-location-through-python-and-selenium-using-chr/43789674#43789674
```

Comment:
Can you format the changes so the lines don't exceed 100 characters? This way they are consistent with the rest of requestium.

You can use services like goo.gl for shortening the url.